Description: fix ident in info4forum
 .
 handylinux-desktop (2.5.1-2) jessie; urgency=medium
 .
   * ajout des sources.list dans info4forum
Author: team handylinux <dev@handylinux.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- handylinux-desktop-2.5.1.orig/info4forum
+++ handylinux-desktop-2.5.1/info4forum
@@ -60,7 +60,7 @@ class Info4forum():
         for sourcefile in os.listdir('/etc/apt/sources.list.d'):
             if sourcefile.endswith('.list'):
                 with open(os.path.join('/etc/apt/sources.list.d',sourcefile), 'r') as s:
-                sources += "/etc/apt/sources.list.d/{}\n{}".format(sourcefile,s.read())
+                    sources += "/etc/apt/sources.list.d/{}\n{}".format(sourcefile,s.read())
 
         t = """    Informations sur {}
 Cette page vous permettra de transmettre des informations à propos de votre système pour obtenir de l'aide sur le forum HandyLinux : le texte sera pré-formaté, il vous suffit de "coller" sur le forum.
